authentication-guides/jira/README.md

```diff
@@ -0,0 +1,267 @@
+
+[This purple chat](https://developer.moveworks.com/creator-studio/purple-chat-builder/?workspace=%7B%22title%22%3A%22My+Workspace%22%2C%22mocks%22%3A%5B%7B%22id%22%3A1636%2C%22title%22%3A%22Mock+1%22%2C%22transcript%22%3A%7B%22settings%22%3A%7B%22colorStyle%22%3A%22LIGHT%22%2C%22startTime%22%3A%2211%3A43+AM%22%2C%22defaultPerson%22%3A%22GWEN%22%2C%22editable%22%3Atrue%2C%22botName%22%3A%22%22%2C%22botImageUrl%22%3A%22%22%7D%2C%22messages%22%3A%5B%7B%22from%22%3A%22USER%22%2C%22text%22%3A%22%3Cp%3EHi%2C+can+you+please+reset+my+Azure+MFA%3F%3Cbr%3E%3C%2Fp%3E%22%7D%2C%7B%22from%22%3A%22BOT%22%2C%22text%22%3A%22%3Cp%3EDo+you+want+to+disable+all+your+Azure+MFA+registrations%3F%3Cbr%3E%3C%2Fp%3E%22%2C%22cards%22%3A%5B%7B%22title%22%3A%22%3Cp%3E%3C%2Fp%3E%22%2C%22text%22%3A%22%3Cp%3E%3C%2Fp%3E%22%2C%22buttons%22%3A%5B%7B%22style%22%3A%22PRIMARY%22%2C%22text%22%3A%22Yes%22%7D%2C%7B%22text%22%3A%22No%22%7D%5D%7D%5D%7D%2C%7B%22from%22%3A%22USER%22%2C%22text%22%3A%22%3Cp%3EYes%3Cbr%3E%3C%2Fp%3E%22%7D%2C%7B%22from%22%3A%22ANNOTATION%22%2C%22text%22%3A%22%3Cp%3EInbound+Request+to+Middleware%3A%3Cbr%3E%3Cbr%3E%7B%3Cbr%3E%5C%22email%5C%22%3A+%5C%22gwen%40moveworks.ai%5C%22%3Cbr%3E%7D%3Cbr%3E%3Cbr%3EProcess%3A%3Cbr%3E1.+Establish+a+connection+to+Azure+MFA+Services%3Cbr%3E2.+List+all+MFA+devices+for+the+user%3Cbr%3E3.+Delete+each+MFA+device%2C+this+will+clear+the+user%27s+devices.%3Cbr%3E%3Cbr%3EOutbound+Response%3A%3Cbr%3E%7B+%5C%22status%5C%22%3A+%5C%22ok%5C%22+%7D%2C+200%3Cbr%3E%3C%2Fp%3E%22%7D%2C%7B%22from%22%3A%22BOT%22%2C%22text%22%3A%22%3Cp%3EI+have+successfully+reset+your+MFA%3Cbr%3E%3C%2Fp%3E%22%7D%5D%7D%7D%5D%2C%22botSettings%22%3A%7B%22name%22%3A%22%22%2C%22imageUrl%22%3A%22%22%7D%7D) shows the experience we are going to build.
+
+![purple-chat.png](Use%20Case%20Guide%20JIRA%20Approvals%20f1ce3fe611e644a99a97f00cfb719fbb/purple-chat.png)
+
+**This use case is made up of:**
+
+✨ **Triggers:** Initiated by JIRA
+
+🤲 **Slots**: None
+
+🏃‍♂️ **Actions: None - Link to JIRA Approval record**
+
+📚 **Guidelines:** None
+
+Based on the needs of this use case, we should build a [Actionable Updates](https://developer.moveworks.com/creator-studio/design-patterns/dp-10).
+
+## API Research
+
+There’s only 1 API needed to build this use case - Which is the Moveworks Events API
+
+### API #1: The Moveworks Events API
+
+Since most of the work is actually done in JIRA Automations, the only API call we will be making is to the Moveworks Events API
+
+![Screenshot 2024-01-09 at 5.20.18 PM.png](Use%20Case%20Guide%20JIRA%20Approvals%20f1ce3fe611e644a99a97f00cfb719fbb/Screenshot_2024-01-09_at_5.20.18_PM.png)
+
+1. [Moveworks API](https://developer.moveworks.com/openapi/reference/#operation/sendMessageForEvent)**:** An API to send proactive events to a Moveworks user
+2. [JIRA Retrive Issue API](https://developer.atlassian.com/cloud/jira/platform/rest/v2/api-group-issues/#api-rest-api-2-issue-issueidorkey-get)**:** A JIRA API to retrieve a given issue and fields
```

Comment:
Why is this referenced? We're not using it?

Reply:
I added this API reference for JIRA because you need to use that to identify what fields you need to map in the response @ajaymerchia